src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 20px;
+    height: 16px;
+    background-image: url("../images/star-active.svg");
+    background-repeat: no-repeat;
+    background-origin: border-box;
+    background-position: 2px center;
+  }
+}
+
+.stars--4:nth-child(n + 5) {
+  background-image: url("../images/star.svg");
+}
```

Comment:
What if this product will have other amount of stars? use [@for](https://sass-lang.com/documentation/at-rules/control/for) loop here